nixos/modules/services/networking/frp.nix

```diff
@@ -31,15 +31,13 @@ in
         default = { };
         description = mdDoc ''
           Frp configuration, for configuration options
-          see the example of [client](https://github.com/fatedier/frp/blob/dev/conf/frpc_legacy_full.ini)
-          or [server](https://github.com/fatedier/frp/blob/dev/conf/frps_legacy_full.ini) on github.
+          see the example of [client](https://github.com/fatedier/frp/blob/dev/conf/frpc_full_example.toml)
+          or [server](https://github.com/fatedier/frp/blob/dev/conf/frps_full_example.toml) on github.
         '';
         example = literalExpression ''
```

Comment:
This doesn't really need to be a `literalExpression`, since it doesn't contain anything that could evaluate (e.g. `pkgs`, `lib` or `config`)

Reply:
Ah sorry, not like that. You'll have to remove the `''`s around as well, so that the example is just an attrset.

```nix
example = {
  serverAddr = "x.x.x.x";
  serverPort = 7000;
};
```

Reply:
oh! got it